src/m_menu.c

```diff
@@ -4072,6 +4036,9 @@ setup_menu_t gen_settings3[] = { // General Settings screen3
   {"Screen melt", S_YESNO, m_null, M_X,
    M_Y + general_screen_melt*M_SPC, {"screen_melt"}},
 
+  {"Level Brightness", S_THERMO, m_null, M_X_THRM,
```

Comment:
How about "Extra Level Brightness"?

Reply:
That's what it was called in the Unity port and inter-doom, I think users have become familiar with this name.